platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/snapshot/SignedStateFileWriter.java

```diff
@@ -112,28 +113,23 @@ public static void writeMetadataFile(
      * Write a {@link SignedState} to a stream.
      *
      * @param out         the stream to write to
-     * @param directory   the directory to write to
      * @param signedState the signed state to write
      */
-    private static void writeStateFileToStream(
-            final MerkleDataOutputStream out, final Path directory, final SignedState signedState) throws IOException {
+    private static void writeSignatureSetToStream(final MerkleDataOutputStream out, final SignedState signedState)
+            throws IOException {
         out.write(VERSIONED_FILE_BYTE);
         out.writeInt(FILE_VERSION);
```

Comment:
the signature set should have a separate file version from the state file. and I don't think it needs to VERSIONED_FILE_BYTE, I think that is just a legacy thing

Reply:
Makes sense. I removed it from this file.